data/json/npcs/godco/cook.json

```diff
@@ -1,267 +1,162 @@
-        "topic": "TALK_GODCO_cook_ask_past"
+        "text": "Could I have some food?",
+        "topic": "TALK_GODCO_chef_get_food",
+        "condition": {
+          "and": [
+            { "npc_has_var": "u_met_godco_chef", "type": "general", "context": "meeting", "value": "yes" },
+            { "not": { "u_has_var": "godco_joinee", "type": "general", "context": "meeting" } }
+          ]
+        }
       },
       {
-        "text": "How are things here?",
-        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
-        "topic": "TALK_GODCO_cook_ask_mood"
+        "text": "I heard the joinees could get some food here.",
+        "topic": "TALK_GODCO_chef_get_food_joinee",
+        "condition": {
+          "and": [
+            { "not": { "npc_has_effect": "u_got_meal" } },
+            { "u_has_var": "godco_joinee", "type": "general", "context": "meeting" }
+          ]
+        }
       },
       {
         "text": "Can I do anything for you?",
-        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "condition": { "npc_has_var": "u_met_godco_chef", "type": "general", "context": "meeting", "value": "yes" },
         "topic": "TALK_MISSION_LIST"
+      },
+      {
+        "text": "Just saying hello.  Keep safe.",
+        "topic": "TALK_DONE",
+        "condition": { "npc_has_var": "u_met_godco_chef", "type": "general", "context": "meeting", "value": "yes" }
       }
     ]
   },
   {
+    "id": "TALK_GODCO_chef_idle_chat",
     "type": "talk_topic",
-    "id": "TALK_GODCO_cook_firstmeet",
-    "dynamic_line": "Shouldn't I be the one to ask that question?  I am Simon, I work here as a cook.  Who are you?",
+    "dynamic_line": [ "Okay, shoot.", "Sure, what's up?", "ALright, lets chat." ],
     "responses": [
-      { "text": "I am an unfortunate who really needs something to eat.", "topic": "TALK_GODCO_cook_ask_food" },
-      { "text": "I am just a fellow survivor.", "topic": "TALK_GODCO_cook_1" }
+      { "text": "What's your story?", "topic": "TALK_GODCO_chef_past" },
+      { "text": "How are you holding up?", "topic": "TALK_GODCO_chef_mood" },
+      { "text": "Is there any way I could join the community?", "topic": "TALK_GODCO_chef_join" },
+      { "text": "I've changed my mind, forget about it.", "topic": "TALK_NONE" },
+      { "text": "On second thought, I gotta go.  See ya, <name_g>.", "topic": "TALK_DONE" }
     ]
   },
   {
+    "id": "TALK_GODCO_chef_intro",
     "type": "talk_topic",
-    "id": "TALK_GODCO_cook_place",
-    "dynamic_line": "This is New England Church Community.  A self-sufficient group formed by churchgoers.  We try to help fellow survivors as much as we can.  Enjoy your stay here!",
+    "dynamic_line": [
+      "This is the New England Community Church, I should say it's one of their old retreats.  I'm Simon, the head chef around here."
```

Comment:
Did the church have multiple compounds?

Reply:
They had a church in one of the cities, and 1-2 charity-related places. To raise money, they probably worked with other organizations that ran retreats; likely how they came to own their current run, as a renovation to later sell. 
At the current time, they only use the retreat.